scipy/stats/tests/test_stats.py

```diff
@@ -4425,6 +4424,15 @@ def test_basic(self):
         F, p = stats.f_oneway([0,2], [2,4])
         assert_equal(F, 2.0)
 
+    @pytest.mark.parametrize('ndim', (1, 2, 3))
+    def test_ndim(self, ndim):
+        a, b = np.arange(0, 3, 2), np.arange(2, 5, 2)
+        for _ in range(ndim - 1):
+            a, b = a[:, np.newaxis], b[:, np.newaxis]
+        F, p = stats.f_oneway([0,2], [2,4])
```

Comment:
f_oneway(a, b) ?

Reply:
Indeed, and it shows that it's actually operating along the first dim, not the raveled array. So the fix here is not going to be correct